lib/puppet/provider/augeasprovider/default.rb

```diff
@@ -42,8 +41,6 @@ def self.supported?(feature)
     case feature
     when :regexpi
       Puppet::Util::Package.versioncmp(aug_version, '1.0.0') >= 0
-    when :post_resource_eval
```

Comment:
I debated keeping this in, since perhaps other augeasproviders use this. However, I didn't check if that's the case.